x/claims/migrations/v2/migration_test.go

```diff
@@ -0,0 +1,49 @@
+package v2_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/cosmos/cosmos-sdk/testutil"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+	"github.com/stretchr/testify/require"
+	"github.com/tharsis/ethermint/encoding"
+	"github.com/tharsis/evmos/v2/app"
+	v2 "github.com/tharsis/evmos/v2/x/claims/migrations/v2"
+	claimstypes "github.com/tharsis/evmos/v2/x/claims/types"
+)
+
+func TestStoreMigration(t *testing.T) {
+	encCfg := encoding.MakeConfig(app.ModuleBasics)
+	claimsKey := sdk.NewKVStoreKey(claimstypes.StoreKey)
+	tClaimsKey := sdk.NewTransientStoreKey(fmt.Sprintf("%s_test", claimstypes.StoreKey))
+	ctx := testutil.DefaultContext(claimsKey, tClaimsKey)
+	paramstore := paramtypes.NewSubspace(
+		encCfg.Marshaler, encCfg.Amino, claimsKey, tClaimsKey, "claims",
+	)
+	require.False(t, paramstore.HasKeyTable())
```

Comment:
not sure if the subspace from before the migration actually has a param key table defined (from the previous version, without the new parameters). If so, we'd need to update the subspace?